docs/api/session.md

```diff
@@ -980,12 +980,17 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
          and this is set to `true`, then local playback of audio will not be muted (e.g. using `MediaRecorder`
          to record `WebFrameMain` with this flag set to `true` will allow audio to pass through to the speakers
          while recording). Default is `false`.
+* `opts` Object (optional)
+  * `useSystemPicker` Boolean - true if a user wants to use the native system picker _Experimental_
```

Comment:
Does the old behavior work in MacOS 15+? Would it make sense to enable this by default in those versions?

Reply:
It does work, yes - the user just receives a warning. Using the new system picker may require a bit of reworking by certain apps, so I was erring on the side of leaving it off by default unless users wanted to explicitly turn it on. We can flip that switch in later versions 👍

Reply:
Would it be worth it to denote in the docs that users will have to re-implement their UI if they choose to use the native system picker?

Reply:
We can! The users would actually not have to re-implement any new UI, which is nice - they'd just have to make sure they're handling the system picker correctly :) I'll address that in the docs.